homework1/tests/task04.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from homework1.task04 import check_sum_of_four
+
+
+@pytest.mark.parametrize(
```

Comment:
If you have only one test case, you shouldn't use `parametrize`, just hardcoded values in the test body. But I recomend to check more edge cases here.

Reply:
👌